helm-chart/examples/gke-ephemeral/scripts/bootstrap.sh

```
@@ -0,0 +1,97 @@
+}
+
+install_tofu() {
+  if command -v tofu >/dev/null 2>&1 && tofu --version 2>/dev/null | grep -q "v${TOFU_VERSION}"; then
+    log "OpenTofu v${TOFU_VERSION} already installed: $(command -v tofu)"
+    return
+  fi
+  log "Installing OpenTofu v${TOFU_VERSION} to ${BIN_DIR}/tofu"
+  local url="https://github.com/opentofu/opentofu/releases/download/v${TOFU_VERSION}/tofu_${TOFU_VERSION}_${OS_ARCH}.tar.gz"
+  local tmp
+  tmp="$(mktemp -d)"
+  trap "rm -rf '${tmp}'" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)